BULK_DIVISIONS_DOCUMENTATION.md

```diff
@@ -0,0 +1,159 @@
+# Bulk Division Creation Feature
+
+## Overview
+
+The bulk Division creation feature allows administrators to quickly create multiple divisions for a season in a single form submission. This feature integrates seamlessly with the existing admin interface.
+
+## Components Added
+
+### 1. DivisionBulkCreateForm
+- **Location**: `tournamentcontrol/competition/forms.py`
+- **Purpose**: Simplified form for division creation with sensible defaults
+- **Fields**: 
+  - `title` (required) - Division name
+  - `short_title` (optional) - Shorter name for display
+  - `copy` (optional) - Public notes about the division
+  - `draft` (optional) - Whether the division is in draft mode
+
+**Automatic Defaults Applied:**
+- `points_formula` = "3*win + 1*draw"
+- `forfeit_for_score` = 5  
+- `forfeit_against_score` = 0
+- `include_forfeits_in_played` = True
+
+### 2. DivisionBulkCreateFormSet
+- **Location**: `tournamentcontrol/competition/forms.py`
+- **Purpose**: Formset for creating multiple divisions at once
+- **Features**:
+  - Inline formset based on Season model
+  - Automatic order assignment for divisions
+  - 3 empty forms by default (`extra=3`)
+  - No deletion allowed in bulk creation mode
+
+### 3. Admin Integration
+- **Location**: `tournamentcontrol/competition/admin.py`
+- **Method**: `bulk_divisions()`
+- **URL Pattern**: `<int:season_id>/bulk-divisions/`
+- **Access**: From Season admin interface
+
```

Comment:
Per other comment the URL should be moved to `<int:season_id>/division/bulk/`.

Further, the view should always be empty - it should not include any existing Divisions, and would add any successful new entries.

Because these are ordered items, if there are existing objects the first newly added form would need to set the `order` to N+1, and so on for extra forms.